### PR TITLE
Map 86

### PR DESCRIPTION
**Pull request by hkf-tech**

### CODE REVIEW

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIGetDraftApprenticeshipOverlapOptions.cs [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            _fixture.VerifyUserRedirectedTo("EmployerNotified");
+        }
+
+        [Test]
+        public async Task AndWhenWhenUserSelectsToContactTheEmployer()
+        {
+            await _fixture.SetupStartDraftOverlapOptions(OverlapOptions.ContactTheEmployer).DraftApprenticeshipOverlapOptions();
+            _fixture.VerifyOverlappingTrainingDateRequestEmail_IsNotSent();
+            _fixture.VerifyUserRedirectedTo("Details");
+        }
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIGetDraftApprenticeshipOverlapOptions.cs:53]: Missing tests for if the user selects to "Add the apprentice later"

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIPostDraftApprenticeshipOverlapOptions.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+            _fixture.VerifyUserRedirectedTo("EmployerNotified");
+        }
+
+        [Test]
+        public async Task AndWhenWhenUserSelectsToContactTheEmployer()
+        {
+            await _fixture.SetupStartDraftOverlapOptions(OverlapOptions.ContactTheEmployer).DraftApprenticeshipOverlapOptions();
+            _fixture.VerifyOverlappingTrainingDateRequestEmail_IsNotSent();
+            _fixture.VerifyUserRedirectedTo("Details");
+        }
```
Comment (cofaulco) [inline src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIPostDraftApprenticeshipOverlapOptions.cs:32]: Missing test for when the user selects to "Add the apprentice later"

**Review by cofaulco — CHANGES_REQUESTED** [review]
couple of comments for missing test scenarios

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIGetDraftApprenticeshipOverlapOptions.cs (hunk 1 above)
Comment (kola-samuel) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIGetDraftApprenticeshipOverlapOptions.cs:53]: Test Created

Comment (kola-samuel) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIGetDraftApprenticeshipOverlapOptions.cs:53]: Done

File: src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIPostDraftApprenticeshipOverlapOptions.cs (hunk 2 above)
Comment (kola-samuel) [reply src/SFA.DAS.ProviderCommitments/SFA.DAS.ProviderCommitments.Web.UnitTests/Controllers/OverlappingTrainingDateRequestControllerTests/WhenIPostDraftApprenticeshipOverlapOptions.cs:32]: done

**Review by kola-samuel — APPROVED** [review]

**Pull request merged**